src/Sylius/Component/Attribute/AttributeType/ChoiceAttributeType.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Attribute\AttributeType;
+
+use Sylius\Component\Attribute\Model\AttributeValueInterface;
+use Symfony\Component\Validator\Context\ExecutionContextInterface;
+
+/**
+ * @author Saša Stamenković <umpirsky@gmail.com>
+ */
+class ChoiceAttributeType implements AttributeTypeInterface
+{
+    const TYPE = 'choice';
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getStorageType()
+    {
+        return AttributeValueInterface::STORAGE_TEXT;
```

Comment:
@pjedrzejewski If we use same attribute type for single and multiple select, they will be both stored using JSON storage? Maybe we can add configuration to `public function getStorageType(array $configuration)` and return storage type based on `$configuration['multiple']`?

Also, what is JSON storage type? Not same as doctrine array?